horreum-web/package.json

```diff
@@ -27,11 +26,12 @@
   "luxon": "3.3.0",
   "moment": "2.30.1",
   "monaco-editor": "0.54.0",
+  "oidc-client-ts": "^3.3.0",
```

Comment:
we can remove the `keycloak-js` dependency :)

Reply:
+1 forgot to remove it